JavaAssignment/src/ravi/CircleRectangle.java

```diff
@@ -0,0 +1,24 @@
+package ravi;
+
+public class CircleRectangle {
+
+	void circleArea(float radius) {
```

Comment:
Only expected the comment before all the methods & class you have written, the rest of the code is very good & clean.